comments/filters.py

```diff
@@ -42,10 +42,10 @@ class ModerationFilter(SimpleListFilter):
 
     def lookups(self, request, model_admin):
         return [
-            (CommentModeration.CommentModerationState.UNMODERATED, 'Unmoderated'),
-            (CommentModeration.CommentModerationState.APPROVED, 'Approved'),
-            (CommentModeration.CommentModerationState.REJECTED, 'Rejected'),
-            (CommentModeration.CommentModerationState.UNSURE, 'Unsure'),
+            (CommunityCommentModeration.State.UNMODERATED, 'Unmoderated'),
+            (CommunityCommentModeration.State.APPROVED, 'Approved'),
+            (CommunityCommentModeration.State.REJECTED, 'Rejected'),
+            (CommunityCommentModeration.State.UNSURE, 'Unsure'),
```

Comment:
We have this in two separate places, and it feels like they will have to be synchronised with each other. This suggest that this should be defined in one place and referenced where it needs to be used.

Isn't the second item in these tuples just the same as the second item `State` itself e.g.:
```python
UNMODERATED[1] == 'Unmoderated'
```